### ISSUE TITLE
When I run `lmql chat` I get an asset not found error

### ISSUE THREAD

**Issue by jbrukh** [eth-sri/lmql#241]
I get the following error when I run `lmql run test.lmql` on one of the test chatbots:

```
% lmql chat test.lmql 
🤖 Your LMQL chatbot is waiting for you at http://localhost:8089
Error handling request
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
  File "/opt/homebrew/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/opt/homebrew/lib/python3.10/site-packages/aiohttp/web_urldispatcher.py", line 195, in handler_wrapper
    result = old_handler(request)
  File "/opt/homebrew/lib/python3.10/site-packages/lmql/lib/chat/chatserver.py", line 15, in handle
    index = open(os.path.join(PROJECT_DIR, 'assets/index.html')).read()
FileNotFoundError: [Errno 2] No such file or directory: '/opt/homebrew/lib/python3.10/site-packages/lmql/lib/chat/assets/index.html'
```

Indeed, when checking this directory, here's the listing, which doesn't contain `assets/`:

```
% ls /opt/homebrew/lib/python3.10/site-packages/lmql/lib/chat/
__init__.py	__main__.py	__pycache__	chatserver.py	output.py
```

Any advice appreciated.

**Comment by lbeurerkellner** [comment]
Thanks for reporting this. It seems the assets were accidentally excluded from the packaging process. You can get a copy of the assets from https://github.com/eth-sri/lmql/tree/main/src/lmql/lib/chat/assets, and place it in the indicated location as a temporary workaround.

I will make sure to publish a bugfix release soon, which will include the chat assets in the pypi version. Sorry about that.

**Comment by lbeurerkellner** [comment]
We just published `0.7.3` which now ships with the chat web UI assets.
